src/web/modules/stripe/model/person.model.ts

```diff
@@ -1,11 +1,13 @@
-function stripeTestResponsiblePersonDetails () {
+import Stripe from "stripe";
+
+export function stripeTestResponsiblePersonDetails (): Stripe.PersonCreateParams {
   return {
     first_name: 'Jane',
     last_name: 'Doe',
     dob: {
-      day: '01',
-      month: '01',
-      year: '1901'
+      day: 1,
+      month: 1,
+      year: 1901
```

Comment:
Are we sure these are equivalent? I suspect so

Reply:
Yeh I tested with the test Stripe API key and it used the expected date